minimesos/src/main/java/com/containersol/minimesos/mesos/MesosSlave.java

```diff
@@ -108,14 +117,21 @@ protected CreateContainerCmd dockerCommand() {
 
     public static ArrayList<Integer> parsePortsFromResource(String resources) throws Exception {
```

Comment:
Please, javadoc the purpose of this method. Could you also add a couple of unit tests, which demo that purpose